src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-data-info/rgw-sync-data-info.component.spec.ts

```diff
@@ -22,4 +24,51 @@ describe('RgwSyncDataInfoComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should display "Up to Date" badge when zone is up to date', () => {
+    component.zone = {
+      timestamp: null
+    };
+    fixture.detectChanges();
+    const upToDateBadge = fixture.debugElement.query(By.css('.badge-success'));
+    expect(upToDateBadge).toBeTruthy();
+    expect(upToDateBadge.nativeElement.textContent).toEqual('Up to Date');
```

Comment:
```suggestion
    const upToDateBadge = fixture.debugElement.query(By.css('.badge-success'));
    expect(upToDateBadge.nativeElement.textContent).toEqual('Up to Date');
```
Is ` expect(upToDateBadge).toBeTruthy();` even needed when we are later on checking `.toEqual()`? 
I guess it kinda helps to avoid throwing an error later on, but does that even matter?

Reply:
I think there's no harm in keeping `expect(upToDateBadge).toBeTruthy();`, it just adds more validity to the tests.